javascript/i18n/phonenumbers/phonemetadata.pb.js

```diff
@@ -2185,23 +2338,23 @@ i18n.phonenumbers.NumberFormat.prototype.getDescriptor = function() {
         type: String
       }
     };
-    i18n.phonenumbers.NumberFormat.descriptor_ =
+    i18n.phonenumbers.NumberFormat.descriptor_ = descriptor =
```

Comment:
I just copied from the generated file :) I think it's to make it less verbose at line 2192, it's just doing an alias.